docs/sql-ref-syntax-aux-show-partitions.md

```diff
@@ -18,5 +18,96 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+The `SHOW PARTITIONS` statement is used to list partitions of a table. An optional
+partition spec may be specified to return the partitions matching the supplied
+partition spec.
+
+### Syntax
+{% highlight sql %}
+SHOW PARTITIONS table_identifier [partition_spec]
```

Comment:
hello, actually i followed `DESCRIBE TABLE` for this. I agree that we should try and stay consistent. I actually like it this way because its more clear to describe the parameter in the section below with its full syntax. However, if there is a consensus, i will change it.

Reply:
I slightly prefer to have the full syntax at the syntax section without having a second layer of sub-syntax at the parameters section. 
@srowen Do you have a preference?
I took a look of SHOW TABLE, ALTER TABLE, TRUNCATE TABLE, the syntax partition_spec is not  consistent among them, either. For the most common parameters, such as partition_spec, table_name, database_name, etc. we need to have an agreed syntax, and then use the same syntax for all the commands. I guess I will need to spend some time to take care of the consistency.

Reply:
@huaxingao Sure.. lets hear from Sean. If required, we could fix the consistencies in this PR.

Reply:
Consistency is best. What's most prevalent in the docs we have? either way, whatever is more desirable and clearer, yes we can do that in a follow up PR to standardize.